### ISSUE TITLE
Link example doesn't work

### ISSUE THREAD

**Issue by nathanhack** [marwan-at-work/vecty-router#9]
Hey there was looking at the example on the README about links

```
func (c *component) Render() vecty.ComponentOrHTML {
	return elem.Span(
		router.Link("/my/route", "click here", nil),
	)
}
```

I assume that the README is just out of date? and should be: `router.Link(ni.Route, "", router.LinkOptions{}),`

I'm just looking for clarification either way; and I can put in a PR to update things if that is helpful.

**Comment by marwan-at-work** [comment]
@nathanhack good catch. Yes, `nil` is incorrect here. Thank you for offering to contribute and I look forward to it :)

**Comment by nathanhack** [comment]
Fixed by #10
